source/user/FAQ.md

```diff
@@ -9,23 +9,23 @@ Every effort has been made to make this FAQ as informative as possible; if you h
 
 ### What is GhostBSD?
 
-Briefly, GhostBSD is a UNIX®-like operating system. It works on AMD64 and Intel® platforms. GhostBSD is based on FreeBSD, which is based on U.C. Berkeley’s “4.4BSD-Lite” release, with some “4.4BSD-Lite2” enhancements. It is also based indirectly on William Jolitz’s port of U.C. Berkeley’s “Net/2” to the i386, known as “386BSD”, though very little of the 386BSD code remains. At this time, GhostBSD provides MATE desktop as default version, and a community release with XFCE desktop.
+Briefly, GhostBSD is a UNIX®-like operating system. It works on AMD64 and Intel® platforms. GhostBSD is based on FreeBSD, which is based on U.C. Berkeley’s “4.4BSD-Lite” release, with some “4.4BSD-Lite2” enhancements. It is also based indirectly on William Jolitz’s port of U.C. Berkeley’s “Net/2” to the i386, known as “386BSD”, though very little of the 386BSD code remains. Currently, GhostBSD provides MATE desktop as the default version and a community release with XFCE desktop.
```

Comment:
Lynne Jolitz also helped quite a bit with 386BSD so we should give her credit too, I think.